schemas/campaigns/activity.schema.json

```diff
@@ -34,6 +34,65 @@
           "type": "string",
           "description": "Activity title."
         },
+        "xdm:brief": {
+          "title": "Brief",
+          "type": "string",
+          "description": "An HTML brief describing the activity."
+        },
+        "xdm:thumbnail": {
+          "title": "Thumbnail",
+          "type": "string",
+          "description": "Thumbnail image representing the activity."
+        },
```

Comment:
It seems like this should be a generalized image field that may contain multiple fields, including the thumbnail. It seems like this is something that should be in XDM because it is generally useful.

Reply:
Yes, this is a good idea. Currently, we do two things:
- For campaigns (experiences), users can pick and choose an image. Customers actually use this feature and choose their own images
- For deliveries (activities), we generate the thumbnails from the email content itself.

I think it'd be good to have different renditions for those images, i.e. generate different sizes/form factors, but this is less important.
Is this what you had in mind (having multiple renditions)? Or did you mean actually having multiple images? I'm good either way, but we'll not change Campaign, i.e. we'll only send the URL of one thumbnail. But it's ok if engage generates and manages multiple renditions too.